DLBlob/index.js

```diff
@@ -55,8 +55,15 @@ function getDlBlobMessages(dlblobText) {
     const parsedBlob = JSON.parse(dlblobText);
 
     if (Array.isArray(parsedBlob) && parsedBlob[0].errorSample) {
-        return parsedBlob[0].messages;
-    } else {
+        var blobMessages = parsedBlob[0].messages;
+        if (blobMessages.length < 10){
```

Comment:
Are you sure 10 is small enough? Maybe we should start with 1?
Could you please refactor `10` into a gloabl constant, like https://github.com/alertlogic/ehub-collector/blob/v1/Master/index.js#L19